src/containers/Repo/Grader/GraderPage.js

```diff
@@ -67,6 +80,10 @@ export default class GraderPage extends Component {
     </Tooltip>);
   }
 
+  close = () => {
+    this.setState({ showModal: false });
```

Comment:
You need to set submitted back to false in the redux store. You need to create a action in the grade.js to help change the submitted back to false when they close the dialog so that if they do try to submit again it will show the submission dialog again